tests/integration/transaction_freeze_e2e_test.py

```
@@ -0,0 +1,148 @@
+    assert unsigned_bytes is not None
+
+    tx2 = Transaction.from_bytes(unsigned_bytes)    
+    assert tx2 is not None
+
+    tx2.sign(executor_key)
+    receipt = tx2.execute(executor_client)
+
+    assert receipt.status == ResponseCode.SUCCESS, "Transaction must execute successfully"
+
+@pytest.mark.integration
+def test_transaction_with_secondary_client_can_execute_successfully(env):
```

Ruff (0.14.10)

Redefinition of unused `env` from line 10

(F811)